src/Domain/Insights/Insight.php

```diff
@@ -29,10 +35,30 @@ abstract class Insight implements InsightContract
     {
         $this->collector = $collector;
         $this->config = $config;
+        $this->excludedFiles = [];
+
+        /** @var array<string> $exclude */
+        $exclude = $config['exclude'] ?? [];
+        if (count($exclude) > 0) {
+            $this->excludedFiles = Files::find(
+                (string) getcwd(),
```

Comment:
How does this act in case where `getcwd` returns false?

Reply:
I tried and it throw a DirectoryNotFoundException. 

I can add a check on $basedir in Files::find and return an empty array if it's not exist to avoid throw an exception ? WDYT

Reply:
Hmm whatever you prefer here tbh. I don't really care that much how it is handled :D